charts/prometheus/templates/sts.yaml

```diff
@@ -203,7 +203,7 @@ spec:
           volumeMounts:
             - name: config-volume
               mountPath: /etc/config
-            - name: storage-volume
+            - name: {{ default "storage-volume" .Values.server.persistentVolume.statefulSetNameOverride "" }}
```

Comment:
Looks good, thank you. I'd suggest, though, to remove the third argument to the `default` function as the function is happy with just two args.

Reply:
@zeritti I've changed it over to a pipe on the original code, and wrapped it in a more defensive if statement if for whatever reason someone passes in `.server.persistentVolume.enabled=false` and  sets `.server.persistentVolume.statefulSetNameOverride`

Reply:
Thank you, looks good.